src/Microsoft.Data.SqlClient/src/Microsoft/Data/StreamExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿#if NETFRAMEWORK
+
+namespace System.IO
```

Comment:
Namespaces should match the location of the file. Also, I generally believe adding code to System namespaces should be avoided since it leads to assumptions that the code is publicly document when it isn't.